media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine"
+HOMEPAGE="http://www.yafaray.org"
+SRC_URI="https://github.com/YafaRay/Core/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/YafaRay/Blender-Exporter/archive/v${PV}.tar.gz -> ${PN}-blender-exporter-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+blender debug +fastmath +fasttrig jpeg opencv openexr png +python tiff truetype"
+
+RDEPEND="dev-libs/boost
+	dev-libs/libxml2
+	sys-libs/zlib
+	blender? ( media-gfx/blender )
+	jpeg? ( virtual/jpeg:0 )
+	opencv? ( >=media-libs/opencv-3.1.0[openexr?] )
+	openexr? ( >=media-libs/openexr-2.2.0 )
+	png? ( media-libs/libpng:= )
+	python? ( ${PYTHON_DEPS} )
+	tiff? ( media-libs/tiff:0 )
+	truetype? ( media-libs/freetype )"
+DEPEND="${RDEPEND}
+	dev-lang/swig"
+
+S="${WORKDIR}/Core-${PV}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i -e "s/@YAFARAY_BLENDER_EXPORTER_VERSION@/v${PV}/" "${WORKDIR}/Blender-Exporter-${PV}/__init__.py" || die
+
+	eapply "${FILESDIR}/${P}-respect-cflags.patch"
+	pushd "${WORKDIR}/Blender-Exporter-${PV}" || die
+	eapply "${FILESDIR}/${P}-blender-exporter-paths.patch"
+	popd || dir
+	eapply_user
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DYAF_LIB_DIR=$(get_libdir)
+		-DWITH_YAF_PY_BINDINGS=$(usex python)
+		-DWITH_YAF_RUBY_BINDINGS=OFF
+		-DBLENDER_ADDON=OFF # addon is a separate package called blender-exporter
+		-DCMAKE_SKIP_RPATH=ON # NULL DT_RUNPATH security problem
+		-DDEBUG_BUILD=$(usex debug)
```

Comment:
This apparently only alters default build mode. Remove the flag, and remove the code that overrides build mode from `CMakeLists.txt`.

Reply:
```
if(DEBUG_BUILD)
#IGNORE     set(CMAKE_BUILD_TYPE Debug CACHE STRING "Build mode" FORCE)
else(DEBUG_BUILD)
#IGNORE     set(CMAKE_BUILD_TYPE Release CACHE STRING "Build mode" FORCE)
endif(DEBUG_BUILD)
```

Is Gentoo adding the #IGNORE stuff?